update known issues (#355)
appsignal/appsignal-docs: source/support/known-issues.html.md

  - Affected components:
      - AppSignal for Ruby package versions: `v2.4.1` - `v2.5.2`
      - Integrations: Sidekiq.

> - [Compatibility issue with other instrumentation gems](known-issues/gem-instrumentation-compatibility.html)

Reviewer: we should probably move these to the top. Most recent issues listed first.

Author: Ah yes I agree, thanks for pointing this out.

Author: I am thinking I will move the 2 newly added issues to the top

> - [Compatibility issue with bugsnag](known-issues/bugsnag-compatibility.html)
>   - Symptom: It produces system stack errors when running rake tasks.

Reviewer: ```suggestion
- [Compatibility issue with other instrumentation gems](known-issues/gem-instrumentation-compatibility.html)
  - Symptom: Apps produce "SystemStackErrors" when other instrumentation gems are loaded, such as http_logger, bugsnag, datadog and potentially others.
```

It's not just Rake, it's all the instrumentations we have and other gems have too.

  - Affected components:

>       - AppSignal for Ruby package versions: `v2.9.17` - `v2.11.0`

Reviewer: I think every version is affected as we never have instrumented it differently.
```suggestion
      - AppSignal for Ruby package versions: `0.0.x` - most recent
```

- [Puma phased restart are broken](known-issues/puma-phased-restart.html)
  - Symptom: Puma phased restart are broken.
  - Affected components:

>       - AppSignal for Ruby package versions: `v2.9.17`

Reviewer: ```suggestion
      - AppSignal for Ruby package versions: `v2.9.17` - most recent
```
